graph/config/cytoscape/cytoscape.go

```diff
@@ -275,7 +277,12 @@ func addNodeTelemetry(n *graph.Node, nd *NodeData) {
 				if protocolTraffic.Rates == nil {
 					protocolTraffic.Rates = make(map[string]string)
 				}
-				protocolTraffic.Rates[string(r.Name)] = fmt.Sprintf("%.*f", r.Precision, rateVal)
```

Comment:
Clever work.  Although I'd like to see the special handling only for rateVal < 1, otherwise just keep the simple Sprintf.  If you prefer just hide everything in a func, maybe like:

`   protocolTraffic.Rates[string(r.Name)] = rateToString(r.Precision, rateVal)`

Reply:
It's a good way to keep code clean. I have modified it.